cmd/crank/beta/trace/internal/printer/default.go

```diff
@@ -147,6 +148,50 @@ func (p *DefaultPrinter) Print(w io.Writer, root *resource.Resource) error {
 		return errors.Wrap(err, errWriteHeader)
 	}
 
+	err := p.printResourceTree(tw, root, isPackageOrRevision)
+	if err != nil {
+		return errors.Wrap(err, "cannot print resource tree")
+	}
+
+	if err := tw.Flush(); err != nil {
+		return errors.Wrap(err, errFlushTabWriter)
+	}
+
+	return nil
+}
+
+// PrintList implements the Printer interface by prints the resource tree of a list of resources in a
+// human-readable format.
+func (p *DefaultPrinter) PrintList(w io.Writer, root *resource.ResourceList) error {
+	tw := printers.GetNewTabWriter(w)
+
+	if len(root.Items) == 0 {
+		return nil
```

Comment:
rather than returning silently, we may inform users with a message such as 'no resources found for this kind'

Reply:
I thought about checking for an empty list in `trace.go`  and return an error there. Not really nice, because for example, json output should be scriptable and should return an empty list in any case.
So whether that is considered an error is depending on the printer. I believe for the default printer it is fairly reasonable to return with an error as it's otherwise not distinguishable from an unresponsive command.